packages/flutter/lib/src/scheduler/ticker.dart

```diff
@@ -391,6 +395,12 @@ class TickerFuture implements Future<Null> {
 
   /// A future that resolves when this future resolves or throws when the ticker
   /// is canceled.
+  ///
+  /// If this property is never accessed, then canceling the ticker does not
+  /// throw any exceptions. Once this property is accessed, though, if the
+  /// corresponding ticker is canceled, then an exception will be thrown on the
+  /// [Future] returned by this getter, and if that exception is not caught,
+  /// there will be an uncaught exception in the current zone.
```

Comment:
nit: I think it's more common to say "complete with error" than "throw exception on future" ([1](https://api.dartlang.org/stable/1.24.3/dart-async/Future/Future.error.html), [2](https://api.dartlang.org/stable/1.24.3/dart-async/Completer/completeError.html)), although that might have been before `async`/`await` added the ability to `try`/`catch` future errors.

Reply:
fixed